src/mappings.ts

```diff
@@ -1,75 +1,93 @@
 import dynamic from 'next/dynamic';
 
-const pageTopicMap = {
-  TopicProduct: dynamic(() =>
-    import('@src/components/features/ctf-components/ctf-product/ctf-product-gql').then(module => module.CtfProductGql),
-  ),
-  TopicBusinessInfo: dynamic(() =>
-    import('@src/components/features/ctf-components/ctf-business-info/ctf-business-info-gql').then(
-      module => module.CtfBusinessInfoGql,
-    ),
-  ),
-  ComponentProductTable: dynamic(() =>
-    import('@src/components/features/ctf-components/ctf-product-table/ctf-product-table-gql').then(
-      module => module.CtfProductTableGql,
-    ),
-  ),
-};
+// const pageTopicMap = {
```

Comment:
Do we still need this code or can it be removed?

Reply:
No we don't need it for now, will delete it